demisto_sdk/commands/validate/tests/validators_test.py

```diff
@@ -401,11 +403,16 @@ def test_run_all_validations_on_file(self, _, file_path, file_type):
         file_validator.run_all_validations_on_file(file_path, file_type)
         assert file_validator._is_valid
 
-    def test_files_validator_validate_pack_unique_files(self,):
+    def test_files_validator_validate_pack_unique_files__file_validator(self,):
```

Comment:
as a first step, we should keep it since some things in content repo still use file_validator functions. I'll open an issue to deprecate the file validator but this is a different issue.